source/includes/steps-fle-create-data-encryption-key.yaml

```diff
@@ -36,6 +36,19 @@ content: |
              if (err) throw err;
              localMasterKey = data;
            });
+     .. tab::
+        :tabid: python
+
+        .. code-block:: python
+           :emphasize-lines: 4
+
+           from bson import binary
+           
+           path = "./master-key.txt";
+           local_master_key = binary.Binary(open(path, "rb").read(96))
+
```

Comment:
Nitpick: Although the additional line breaks are not rendered, could we limit them to at most, a single blank line?

Reply:
Fixed:

* Ran all the py code through a vscode linter